index.ts

```diff
@@ -49,6 +56,8 @@ async function onEvent(event: PluginEvent, { config, global }: IntercomMeta) {
     )
 }
 
+// product question, maybe for @marcushyett-ph: when a contact doesn't exist, do we maybe want to create it?
```

Comment:
@marcushyett-ph

Reply:
I think it makes sense not to create it - with posthog being the source of truth for all users and only customers you're chatting with as contacts in intercom.